snippets/rust/rust.json

```diff
@@ -136,14 +141,24 @@
     },
     "print": {
         "prefix": "print",
-        "body": ["print!(\"${1}\");"],
+        "body": ["print!(\"${1}\"${2});"],
         "description": "print!(…);"
     },
     "println": {
         "prefix": "println",
-        "body": ["println!(\"${1}\");"],
+        "body": ["println!(\"${1}\"${2});"],
         "description": "println!(…);"
     },
+    "eprint": {
+        "prefix": "eprint",
+        "body": ["eprint!(\"${1}\"${2});"],
+        "description": "eprint!(…);"
+    },
+    "eprintln": {
+        "prefix": "eprintln",
+        "body": ["eprintln!(\"${1}\"${2});"],
```

Comment:
All of the print style snippets should have the double quotes removed. The end user should be able to specify those, if needed, simply by us having the `${1}` argument correctly placed.